backend/note_taking/views/block_views.py

```
@@ -0,0 +1,68 @@
+        return queryset
+
+
+class CreateBlockEndpoint(CreateAPIView):
+    queryset = Block.objects.all()
+    serializer_class = BlockSerializer
+    permission_classes = [IsAuthenticated]
+
+    def perform_create(self, serializer):
+        note = serializer.validated_data["note"]
+        if note.notebook.owner.id != self.request.user.id:
+            raise PermissionDenied("You do not have permission to create blocks in this note")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)